tests/Http/Api/PostApiTest.php

```diff
@@ -125,6 +125,101 @@ public function testCreatingAPost()
         ]);
     }
 
+    /**
+     * Test that a POST request to /posts with an existing post creates an additional new photo post.
+     * This test ensures we are not breaking Gambit when it creates a post for a signup that already has posts.
```

Comment:
A very nitpicky thing- I don't think we need to contextualize these tests to only Gambit flows. This is a good test to have for any other service that will be sending over multiple posts